src/main/java/frc/robot/Constants.java

```diff
@@ -70,4 +70,34 @@ public static final class Swerve {
     public static final Translation2d BL_LOCATION = new Translation2d(-APOTHEM, APOTHEM);
   }
 
+  public final static class Intake {
+    // ~ WIP ~ //
+    public static final double PIVOT_kP = 0;
+    public static final double PIVOT_kI = 0;
+    public static final double PIVOT_kD = 0;
+
+    public static final int INTAKE_MOTOR_ID = 0;
+    public static final int PIVOT_MOTOR_ID = 0;
+    public static final int DIO_CHANNEL_LIMITSWITCH = 0;
+
+    public static final double INTAKE_MOTOR_SPEED = 0;
+    public static final double PIVOT_MOTOR_SPEED = 0;
+    public static final double INTAKE_MOTOR_SPIT_SPEED = 0;
+
+    public static final double ENCODER_POSITION = 0;
+
+    public static final double MAX_VELOCITY = 0;
+    public static final double MAX_ACCELERATION = 0;
+
+    // Pivot set point angles, thought I still don't know what the angles would be.
+    public static final double k_PIVOT_ANGLE_GROUND = 0;
+    public static final double k_PIVOT_ANGLE_SOURCE = 0;
+
+    public static final double TOLERANCE = 0;
+    public static final double DEPLOY_SETPOINT = 0;
+    public static final double STOW_SETPOINT = 0;
+    public static final double NO_VOLTS = 0;
```

Comment:
I think everyone knows no volts is 0 it might not be necessary but ok

Reply:
nuh uh